datascience/utils.py

```diff
@@ -41,3 +43,31 @@ def get_production(countries, start_date, end_date, delta):
                               'production': o['production'].values()})
             df = df.append(p)
     return df
+
+# get_exchange
+
+def fetch_exchange(country_code, t):
+    endpoint = 'http://electricitymap.tmrow.co'
+    url = '%s/v1/state' % endpoint
```

Comment:
I think keeping the CO2 intensity is interesting for that function. We could for instance return an additional column `co2_intensity` in addition to the `power exchanged`. 

What's the problem with running the Co2 model here ? Not sure Python users of Electricity Map will overcharge the servers yet :)